scripts/upgrade

```diff
@@ -89,21 +95,10 @@ ynh_system_user_create --username=$app --home_dir=$final_path
 
 if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
-	ynh_script_progression --message="Upgrading source files..." --weight=7
-
-	# Create a temporary directory
-	tmpdir="$(ynh_smart_mktemp --min_size=10)"
+	ynh_script_progression --message="Upgrading source files..." --weight=1
 
 	# Download, check integrity, uncompress and patch the source from app.src
-	ynh_setup_source --dest_dir="$tmpdir"
-
-	# Backup the config file in the temp dir
-	cp -a "$final_path/config.php" "$tmpdir/config.php"
-
-	# Replace the old ttrss by the new one
-	ynh_secure_remove --file="$final_path"
-	mv "$tmpdir" "$final_path"
-	ynh_secure_remove --file="$tmpdir"
+	ynh_setup_source --dest_dir="$final_path" #--keep="$final_path/config.php"
```

Comment:
Should the `--keep` be actually removed?

Reply:
indeed! the config file syntax as completely changed